common/saber_base.h

```diff
@@ -38,6 +38,13 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(SECONDARY_IGNITION)		\
+    DEFINE_EFFECT(SECONDARY_RETRACTION)		\
+    DEFINE_EFFECT(QUOTE)			\
+    /* Change Alt font sounds */                \
```

Comment:
This comment doesn't explain much, two of the words are already in the ALT_SOUND names.
If you read this (and nothing else) would you understand what it meant?
I don't think it needs a long explanation, but maybe something like:
/* Triggered when we change sets of sounds within the font from one alternative to another. */

Reply:
ok, updated.  although this effect will be what triggers the change for the sets of sounds when it's triggered.  the actual trigger is style / prop specific.